.gitmodules

```diff
@@ -1,17 +1,18 @@
 [submodule "wxWidgets"]
 	path = wxWidgets
-	url = https://github.com/RPCS3/wxWidgets.git
+	url = https://github.com/Zangetsu38/wxWidgets
 	ignore = dirty
+	branch = master
 [submodule "rpcs3-ffmpeg"]
 	path = ffmpeg
-	url = https://github.com/hrydgard/ppsspp-ffmpeg
+	url = https://github.com/bigpet/ppsspp-ffmpeg
```

Comment:
same for this one, let me talk to hrygard again and I'll see if we can't work out a way to get VS2015 support in his repo. There's really no reason to fork it yet.